packages/lib-classifier/src/store/WorkflowStore/Workflow/Workflow.spec.js

```diff
@@ -54,6 +54,8 @@ describe('Model > Workflow', function () {
     it('should have limit_subject_height default to false', function () {
       expect(workflow.configuration.limit_subject_height).to.be.false()
     })
+
+    // need a unit test for multi_image_clone_markers here
```

Comment:
Workflow.spec includes tests for WorkflowConfiguration store, so need something for `multi_image_clone_markers` added here.